CHANGELOG.md

```diff
@@ -24,10 +24,12 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 - Development: Add dummy isolation provider for testing non-conversion-related
   issues in virtualized Windows and MacOS, where Docker can't run, due to the
   lack of nested virtualization ([issue #229](https://github.com/freedomofpress/dangerzone/issues/229))
-- Add support for more MIME types that were previously disregarded ([issue #377](https://github.com/freedomofpress/dangerzone/issues/377))
+- Add support for more MIME types that were previously disregarded ([issue #369](https://github.com/freedomofpress/dangerzone/issues/369))
 
 ### Changed
 
+- Full release under FPF: signing keys have changed from the original developer Micah Lee / First Look Media to FPF's signing keys. Linux packages moved from PackageCloud to FPF's servers
+- [Installation instructions updated](https://github.com/freedomofpress/dangerzone/blob/v0.4.1/INSTALL.md) to reflect change in key owership to FPF
```

Comment:
Noting this is a 404 for now, but resolves with the RC tag and will work once the new tag is live.
https://github.com/freedomofpress/dangerzone/blob/v0.4.1-rc3/INSTALL.md

Reply:
Exactly. Also note that users will not stumble onto this, due to the fact that we will merge it to the `release-0.4.1` branch.